spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature 'adding comments' do
+
+  let!(:sea) {Photo.create(title:'Awesome sea',
+                             image: File.new("spec/features/sea.jpg") )}
+
+  skip 'allows users to leave a comment' do
+    visit '/photos'
+    click_link 'Add a comment'
```

Comment:
This test is failing as the browser is not on the page that we want it to be on. Use launchy to debug problems like this.